gitlab/packages/gitlab/gitlab_stats.js

```diff
@@ -0,0 +1,84 @@
+// jshint esversion: 9
+
+/**
+ * @description null
+ * @param {ParamsType} params list of command parameters
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+
+const axios = require('axios');
+
+async function fetchStats(url) {
+    return await axios.get(url)
+    .then(response => { return response.data; })
+    .catch(error => { return error.response.data; });
+}
+
+async function _command(params, commandText, secrets = {}) {
+  const {
+    repo
+  } = params;
+  
+  // The gitLab api endpoint for viewing information on projects
+  const repoURL = `https://gitlab.com/api/v4/projects/${repo.replace(/\//g, "%2F")}`;
+  
+  // A dictionary of api urls for each statistic
+  const urlKey = {
+    fetches: `${repoURL}/statistics?access_token=${secrets.AcessToken_GitLab}`,

```

Comment:
There needs to be a check to make sure secrets are set by the user. Please make sure wherever you're using `AcessToken_GitLab`, there's a check to make sure its present. See `awsbill` command-set for example on how to throw a message if it isn't available.

Reply:
Those cases are currently handled, I typically prefer to return the APIs error response when a key is not provided. I can personalize the message if that's what you want.